src/dirigera/devices/outlet.py

```diff
@@ -10,8 +11,29 @@
 
 
 class OutletAttributes(Attributes):
+    custom_name: Optional[str] = None
```

Comment:
you can remove all the attributes here that are already defined in the `Attributes` class

Reply:
if you remove them it will still be too many so the linter will complain, just add
`# pylint: disable=too-many-positional-arguments`
after line 13